src/server.rs

```diff
@@ -29,6 +29,19 @@ async fn index(config: web::Data<RwLock<Config>>) -> Result<HttpResponse, Error>
         .landing_page_content_type
         .clone()
         .unwrap_or("text/plain; charset=utf-8".to_string());
+
+    match &config.server.landing_page_file {
+        Some(file) => {
+            let file_contents = fs::read_to_string(file).unwrap_or("".to_string());
+            if !file_contents.is_empty() {
```

Comment:
Let's handle the error gracefully via `?` operator here.

Reply:
Same here. No idea how to use the `?` operator.

I'm more than happy to get help. It took me already hours to figure out how to read a file, so that the program does not panic when the file is not found. In C this is all very easy. In Rust it makes my head hurt.

Reply:
I will read up on the lang reference. Thanks.